content/sensu-core/1.5/changelog.md

```diff
@@ -52,6 +53,33 @@ menu: "sensu-core-1.5"
 - [Core 0.23.1 Release Notes](#core-v0-23-1)
 - [Core 0.23.0 Release Notes](#core-v0-23-0)
 
+## Core 1.5.0 Release Notes {#core-v1-5-0}
+
+Source: [GitHub.com][62]
+
+**September 4, 2018** &mdash; Sensu Core version 1.5.0 has been released
+  and is available for immediate download. Please note the following
+  improvements:
+
+### CHANGES {#core-v1-5-0-changes}
+
+- **IMPORTANT**: Sensu packages now include Ruby 2.4.4. This includes a fix
+  for a bug in Ruby 2.4 that impacted Sensu clients on AIX. Upgrading from
+  releases of Sensu prior to 1.0.0 will require all plugin or
+  extension gems to be re-installed under the new Ruby environment.
+
+- **IMPROVEMENT**: The built-in check dependencies filter lets you
+  reduce noise by only alerting for the root cause of a given failure.
+  Sensu Core 1.5 gives you the option to specify check dependencies by subscription
+  instead of by client, making this useful filter even more flexible.
+  See the [filter reference doc](/sensu-core/1.5/reference/filters#check-dependencies-filter) for more information.
+
+- **BUGFIX**: Improved validation for check results created using the
+  results API and the client socket. Sensu now validates check result
+  attributes with the same methods used to validate check definitions.
+
+- **BUGFIX**: Fixed a bug in the Sensu package that prevented Linux builds
```

Comment:
@amdprophet is this wording correct? I had understood the bug as "caused Linux builds to add a `sensu` user and group to the system regardless of whether or not they existed in LDAP"

Reply:
@cwjohnston your understanding is correct. The way the package scripts were checking to see if a user/group already exists only checked against the local system. We're now using `getent` on Linux which can check against multiple user/group providers.